website/source/docs/providers/aws/r/elb.html.markdown

```diff
@@ -59,6 +62,9 @@ The following arguments are supported:
 * `listener` - (Required) A list of listener blocks. Listeners documented below.
 * `health_check` - (Optional) A health_check block. Health Check documented below.
 * `cross_zone_load_balancing` - (Optional) Enable cross-zone load balancing.
+* `idle_timeout` - (Optional) Adjust Idle Timeout.
+* `connection_draining` - (Optional) Enable connection draining.
+* `connection_draining_timeout` - (Optional) Adjust the timeout for connection draining.
```

Comment:
It would be helpful to be more descriptive here. 

- `idle_timeout`: The time in seconds that the connection is allowed to be idle. Default: _whatever the default is_ 
- `connection_draining` - (Optional) Boolean to enable connection draining.
- `connection_draining_timeout` - (Optional) The time in seconds to allow for connection draining. (or whatever)..